### ISSUE TITLE
Functional Training Loop

### ISSUE THREAD

**Issue by tatiana-boura** [eneribnk/Tasks_plasmAI#3]
Create model, train it on training data and observe training loss progression.
No evaluation ath this stage.
